widget/ui/src/components/I18nManager/I18nManager.tsx

```diff
@@ -16,7 +16,7 @@ import { messages as esMessages } from '../../../../../translations/es';
 import { messages as frMessages } from '../../../../../translations/fr';
 import { messages as jpMessages } from '../../../../../translations/jp';
 
-const DEFAULT_LANGUAGE = 'en';
+export const DEFAULT_LANGUAGE = 'en';
```

Comment:
I think moving this const to `widget/embedded/src/constants/languages.ts` is more appropirate place.

Reply:
Done